doc/docs/stdlib.html

```diff
@@ -182,6 +182,17 @@ <h4 id="strReplace">std.strReplace(str, from, to)</h4>
    yields <code>"I like to surf with my surfboard"</code>. </p>
 
 
+<h4 id="charReplace">std.charReplace(input_str, from_char, to_str)</h4>
+
+<p>Returns a copy of the string in which all occurrences of char <code>from_char</code> have been replaced
+   with string <code>to_str</code></p>
+
+<p>This is ~6x faster than <code>std.strReplace</code> when replacing single characters</p>
```

Comment:
Nit: I would just write that it's "much" faster, not 6x faster. The difference in speed depends on specific benchmark and it will be a pain to maintain as the implementation evolves.

Reply:
updated. thanks